Gemfile

```diff
@@ -1,8 +1,9 @@
 source 'https://rubygems.org'
 
 group :test do
-  gem "asciidoctor", :git => "https://github.com/asciidoctor/asciidoctor.git", :ref => '77091bf96780c738027a1846f4a8a0cc8809b8dd'
+  gem "asciidoctor"
```

Comment:
Question: Maybe we want to maybe pin it to a major version just in case they introduce more breaking changes in, say, 3.0, but I feel like also maybe it's not a big deal.

Reply:
I don't think it's a big deal for this repo, because only the templates are used in Production. However, I think we should probably pin it as you suggest in the Workers Gemfile.